libs/ts-rest/solid-query/package.json

```diff
@@ -34,5 +34,9 @@
   "main": "index.cjs",
   "module": "index.js",
   "types": "./src/index.d.ts",
-  "type": "module"
+  "type": "module",
```

Comment:
Will `module` affect any of the rollup compilation? Want to make sure there are no regressions.

Reply:
This isn't new, I believe we had many tanstack lib issues without it

Reply:
my b, confused the github diff